devito/arguments.py

```diff
@@ -175,51 +212,41 @@ def dtype(self):
 
     @cached_property
     def rtargs(self):
-        if self.size is not None:
-            return []
-        else:
-            size = ScalarArgument("%s_size" % self.name, self, max)
-            return [size]
-
-    @property
-    def decl(self):
-        return self.rtargs[0].decl
+        size = ScalarArgument("%s_size" % self.name, self, max)
+        return [size]
```

Comment:
inlinable

Reply:
The next PR will extend this anyway. Do you really want it inline for a couple of days? :)

Reply:
nope :)